src/core/withExperiment.js

```diff
@@ -69,9 +69,24 @@ export const withExperiment = ({
 
       this.experimentCookie = _cookie.load(this.getCookieName());
 
+      log.info(
```

Comment:
just a quick question: I've read that `this.props` and `props` are supposed to be the same in the constructor but I've ran into an issue where it was actually different so things didn't work as expected. Maybe that was a deeper issue but if I am remembering correctly it was a similar situation where it was ok locally but not on another env. I am wondering if we should log out the different props just to see those..?

Reply:
which props would you like to see here?

Reply:
something like ~

`log.info(this.props)`;

AND

`log.info(props)`

just to make sure they are the same

Reply:
the problem is that `config` and `i18n` are part of each variable, and it is big!

Reply:
oh right. So maybe just the props that we are using then ~

log.info(this.props._cookie);
log.info(this.props.variantA)
log.info(this.props.variantB)

log.info(props._cookie);
log.info(props.variantA);
log.info(props.variantB)

?

Reply:
added!